lib/EnsEMBL/REST/Controller/Info.pm

```diff
@@ -94,7 +94,7 @@ sub species : Local : ActionClass('REST') :Args(0) { }
 
 sub species_GET :Local :Args(0) {
   my ($self, $c) = @_;
-  my $division = $c->request->param('division');
+  my $division = $c->request->param('division') || 'EnsemblVertebrates';
```

Comment:
Something is very up with the strains in the Travis test of info.t

Reply:
We're also leaning toward the  // operator these days, as it isn't fooled by numeric zeroes

Reply:
Changed to //